README.md

```diff
@@ -48,11 +42,15 @@ This project helped us get quickly started with LSP. Since then, we have
 refactored the project's original sources to the point where only a few simple
 case classes remain.
 
-## Related work
+## Alternatives
 
-* [ENSIME](http://ensime.org): a tool for providing IDE-like features to text
-  editors, that
-  [recently added LSP support](https://github.com/ensime/ensime-server/pull/1888)
+* [IntelliJ](https://www.jetbrains.com/help/idea/discover-intellij-idea-for-scala.html):
```

Comment:
"Intellij IDEA"?

Reply:
I guess, you meant Intelli**J**, but I literally copy-pasted the name from title of the linked article 😆 They use these two spelling inconsistently on the same page: 

<img width="994" alt="screen_shot_2018-04-02_at_21_20_32" src="https://user-images.githubusercontent.com/766656/38211944-618c2d76-36bc-11e8-93f4-121fe40ffaf7.png">

I personally couldn't care less, but I think it's funny 😂